common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1568,7 +1586,7 @@ def set(self, attr, val):
         self.__dict__[attr] = val
 
     def __str__(self):
-        return str(self.__dict__)
+        return "LmsModuleSystem{}".format(self.__dict__)
```

Comment:
Shouldn't this be "ModuleSystem"?  Maybe use `self.__class__`.  And why is it `__str__` instead of `__repr__`?  This won't get called for repr.